src/frontend/csrs-portal/src/app/components/welcome-user/welcome-user.component.ts

```diff
@@ -1,15 +1,79 @@
 import { Component, OnInit } from '@angular/core';
+import { map } from 'rxjs/operators';
+import { JsonPipe } from '@angular/common';
+import { AccountService } from 'app/api/api/account.service';
+import { LookupService } from 'app/api/api/lookup.service';
+import { LoggerService } from '@core/services/logger.service';
+import { of } from 'rxjs';
+import { Inject, Injectable, Optional }                      from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams, HttpStatusCode,
+         HttpResponse, HttpEvent, HttpParameterCodec }       from '@angular/common/http';
+import { Observable }                                        from 'rxjs';
+import { AccountFileSummary } from 'app/api/model/accountFileSummary.model';
+import { Router, ActivatedRoute } from '@angular/router';
+import { AppRoutes } from 'app/app.routes';
+
 
 @Component({
   selector: 'app-welcome-user',
   templateUrl: './welcome-user.component.html',
   styleUrls: ['./welcome-user.component.scss']
 })
 export class WelcomeUserComponent implements OnInit {
+  _logger: LoggerService;
+  _reponse: HttpResponse<AccountFileSummary>;
 
-  constructor() { }
+  constructor(@Inject(AccountService) private accountService,
+              @Inject(LoggerService) private logger,
+              @Inject(Router) private router) {}
 
   ngOnInit(): void {
+    this._logger = this.logger;
+
+
+    this.accountService.apiAccountGet('response', false).subscribe({
+      next: (data) => {
+        this._reponse = data;
+        if (this._reponse.status === HttpStatusCode.NotFound) {
+          //redirect to empty application form
+          this.router.navigate(AppRoutes.routePath(AppRoutes.APPLICATIONFORM));
+        }
+        if (this._reponse.status === HttpStatusCode.Ok) {
+          if (this._reponse.body.files === null || this._reponse.body.files.length === 0)
+          {
+            //redirect to empty application from
+            this.router.navigate(AppRoutes.routePath(AppRoutes.APPLICATIONFORM));
+          }
+          else
+          {
+            if (this._reponse.body.files.length > 0)
+            {
+              if (this._reponse.body.files[0].status === 'Draft')
+              {
+                //redirect to Application form with draft application
+                this.router.navigate(AppRoutes.routePath(AppRoutes.APPLICATIONFORM));
```

Comment:
Hey @skiselev1 
Do we still have the DRAFT option as per the new/reduced requirement? If not then please get rid of it.
Also are those statuses like 'Active' case sensitive?

Thanks Sergey for looking!